### PR TITLE
Refactor OutGridView.Gui into GraphicalTools.Gui

### LINKED ISSUE #8: Refactor Gui project to be general purpose
Right now, we have `OutGridView.Gui` which obviously only supplies the GUI side of Out-GridView. 

When we want to add other GUI tools like `Show-Command` and the likes we should refactor & rename `OutGridView.Gui` to `GraphicalTools.Gui` and figure out a way to signal `GraphicalTools.Gui` to show the `Show-Command` GUI instead of the `Out-GridView` GUI. This would allow us to reuse the Avalonia dlls that take up a lot of space.

### PR DESCRIPTION
**Pull request by corbob**
Fixes #8

### CODE REVIEW

File: src/Microsoft.PowerShell.GraphicalTools/OutGridviewCmdletCommand.cs [hunk 1]
```diff
@@ -7,16 +7,16 @@
...
 
-namespace OutGridView.Cmdlet
+namespace GraphicalTools.Cmdlet
 {
     /// Enum for SelectionMode parameter.
     /// </summary>
     [Cmdlet(VerbsData.Out, "GridView", DefaultParameterSetName = "PassThru")]
     [Alias("ogv")]
-    public class OutGridViewCmdletCommand : PSCmdlet
+    public class GraphicalToolsCmdletCommand : PSCmdlet
```
Comment (TylerLeonhardt) [inline src/Microsoft.PowerShell.GraphicalTools/OutGridviewCmdletCommand.cs]: Don't want to change this. We still want this to be Out-GridView.

**Comment by TylerLeonhardt** [comment]
The initial renames are looking good. We'll need some sort of mechanism to tell GraphicalTools.Gui that the data it's being given needs to be rendered in the GridView if Out-GridView was used.

File: src/Microsoft.PowerShell.GraphicalTools/OutGridviewCmdletCommand.cs (hunk 1 above)
Comment (corbob) [reply src/Microsoft.PowerShell.GraphicalTools/OutGridviewCmdletCommand.cs]: Good catch, I'm still wrapping my head around that cmdlets are classes within the namespace and not methods/functions of the overall class.

**Comment by corbob** [comment]
I'm going to close this PR since it's been basically a year since I've looked at it, and issue #101 speaks of moving to MAUI. I suspect that MAUI will take a rearchitecture that would render this PR meaningless.

**Pull request closed without merging**